apps/user-profile/src/app/pages/consents-page/consent-request/consent-request.component.html

```diff
@@ -0,0 +1,24 @@
+<div class="d-flex">
+  <div class="card p-4 mat-elevation-z3 ml-auto mr-auto">
+    <h1 class="page-title">
+      {{ 'CONSENTS.REQUEST.TITLE' | customTranslate | translate }}
+    </h1>
+    <div class="page-subtitle">{{consent.consentHub.name}}</div>
+
+    <div>
+      {{ 'CONSENTS.REQUEST.CONSENT_TEXT_UNSIGNED' | customTranslate | translate }}:
+      <ul>
+        <li *ngFor="let attribute of consent.attributes">
+          {{ attribute.displayName }}
+        </li>
+      </ul>
+    </div>
+    <!--    <mat-checkbox class="minimize-checkbox mb-n2">{{ 'CONSENTS.REQUEST.CHECKBOX' | customTranslate | translate }}</mat-checkbox>-->
+    <button mat-flat-button class="mb-2" color="accent" (click)="grantConsent()">
+      {{ 'CONSENTS.REQUEST.GRANT_CONSENT_BUTTON' | customTranslate | translate }}
+    </button>
+    <button mat-flat-button color="warn" (click)="rejectConsent()">
+      {{ 'CONSENTS.REQUEST.REJECT_CONSENT_BUTTON' | customTranslate | translate }}
+    </button>
```

Comment:
I would prefer for this buttons to not take full width of the card, but rather be in the bottom right corner as usual with "positive" action on the right.